src/sql/workbench/services/accountManagement/browser/accountPickerService.ts

```diff
@@ -53,6 +55,16 @@ export class AccountPickerService implements IAccountPickerService {
 		}
 	}
 
+	public setInitialAccountTenant(account: string, tenant: string): void {
+		if (this._accountPicker) {
+			this._accountPicker.setInitialAccount(account);
+			this._accountPicker.setInitialTenant(tenant);
+		} else {
+			this._logService.error('Account Picker was undefined. Could not set initial account/tenant for firewall dialog.');
+		}
+	}
+
+
```

Comment:
I see in the 'renderAccountPicker' method, provider is hardcoded to public cloud - looks like something can be fixed in another PR. Please open a bug to track the same.

Reply:
https://github.com/microsoft/azuredatastudio/issues/22698 Opened this issue